package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "github-notifier-chrome",
```

Comment:
For `npm` to parse the file, I believe it needs a `name` field. I will check later tonight, on my phone now.